.github/workflows/validate.yml

```diff
@@ -59,21 +71,15 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Install stable toolchain
-        uses: actions-rs/toolchain@v1
+        uses: dtolnay/rust-toolchain@stable
         with:
-          profile: minimal
           toolchain: stable
-          override: true
           components: rustfmt, clippy
 
       - name: Run cargo fmt
-        uses: actions-rs/cargo@v1
-        with:
-          command: fmt
-          args: --all -- --check
+        run: cargo fmt --all -- --check
 
       - name: Run cargo clippy
-        uses: actions-rs/cargo@v1
-        with:
-          command: clippy
-          args: -- -D warnings
+        run: |
+          cargo clippy -- -D warnings
+          cargo clippy --no-default-features --features num -- -D warnings
```

Comment:
@AndrewWestberg was this just for testing? because on main we should cover all features.

Reply:
This runs clippy twice. Once with the default features (gmp), and once with the `num` feature. The code is designed so that you must enable at least one of the features, but cannot enable both at the same time.

Reply:
see `lib.rs`